setup.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import codecs
+from setuptools import setup
+
+
+def read(fname):
+    file_path = os.path.join(os.path.dirname(__file__), fname)
+    return codecs.open(file_path, encoding="utf-8").read()
+
+
+setup(
+    name="pytest-kiwitcms",
+    version="0.1.0",
+    author="Dmitry Dygalo",
+    author_email="dadygalo@gmail.com",
+    maintainer="Dmitry Dygalo",
+    maintainer_email="dadygalo@gmail.com",
+    license="GNU GPL v3.0",
+    url="https://github.com/kiwitcms/pytest-plugin",
+    description="Pytest plugin for Kiwi TCMS",
+    long_description=read("README.rst"),
+    packages=["pytest_kiwitcms"],
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+    install_requires=["pytest>=4.0.0"],
```

Comment:
Can you make this into a `requirements.txt` file, see:
https://github.com/kiwitcms/tap-plugin/blob/master/setup.py#L24

The reason is that we generally like to pin requirements versions and use a service like PyUP.io or Dependabot to let us know about new versions. The benefit of this is that we know exactly which versions of dependencies are used and when a new one is found the bot will open a pull request and we'll know if it works or doesn't.

As much as pytest can try to be backwards compatible there will be time when the plugin will break.

You will also need `tcms-api` as dependency to be able to communicate back to the server. For this we'd generally want to pin the version because there can be different API methods supported.